megamek/src/megamek/server/Server.java

```diff
@@ -3444,7 +3444,7 @@ private void changeToNextTurn(int prevPlayerId) {
         if (prevPlayerId != -1) {
             send(createTurnIndexPacket(prevPlayerId));
         } else {
-            send(createTurnIndexPacket(player.getId()));
+            send(createTurnIndexPacket(player != null ? player.getId() : IPlayer.PLAYER_NONE));
```

Comment:
"unload stranded entities" turn doesn't happen for a specific player, it happens for every player, so the turn's player ID is -1.